common/tokensParser_test.go

```diff
@@ -54,6 +54,16 @@ func TestExtractTokenIDAndNonceFromTokenStorageKey(t *testing.T) {
 		// "-" represents nonce 45 and should not be treated as a separator
 		checkTickerAndNonceExtraction(t, "EGLDMEXF-8aa8b6-", "EGLDMEXF-8aa8b6", 45)
 	})
+
+	t.Run("prefixed tokens", func(t *testing.T) {
+		t.Parallel()
+
+		checkTickerAndNonceExtraction(t, "pref-ALC-1q2w3e", "pref-ALC-1q2w3e", 0)
+		checkTickerAndNonceExtraction(t, "pf1-ALC-1q2w3e", "pf1-ALC-1q2w3e", 0)
+
+		checkTickerAndNonceExtraction(t, "sv1-TKN-1q2w3e4", "sv1-TKN-1q2w3e", 52)
```

Comment:
We also need tests with min/max characters for prefix and min/max characters for ticker, and combination of these.